support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -15,6 +15,18 @@ Next:
     copy_executable: {{ copy_executable | default("None") }}
     submit_script_template: {{ submit_script_template | default("None") }}
     submit: True
+{% elif eccentricity_control %}
```

Comment:
This boolean needs to be passed to this YAML file from `start_inspiral()`. Add a CLI option in `Inspiral.py` and pass the boolean on.